recipes/pblat/meta.yaml

```diff
@@ -7,9 +7,6 @@ package:
 source:
   url: https://github.com/icebert/pblat/archive/{{ version }}.tar.gz
   sha256: 281440d93c4a555aceb8d5dd97573f69f001af43a33a44fa4663742a04adecc3
-  #https://github.com/icebert/pblat/pull/14
-  patches:
-    - patch
 
 build:
   number: 0
```

Comment:
Please increase the build number to 1.

Reply:
Actually, sorry about that - I just saw that the 2.5 version hasn't been merged yet.